tgui/packages/tgui/interfaces/NuclearBomb.js

```diff
@@ -7,7 +7,7 @@ export const NuclearBomb = (props, context) => {
 
   if (!data.extended) {
     return (
-      <Window width={450} height={300}>
```

Comment:
Don't we still need this? Or are we just going with the default size

Reply:
Having poked around the window, didn't see any change in window size
Maybe it's best to delete it altogether

Reply:
Okay, this is for the undeployed nuke, changed